hibernate-core/src/main/java/org/hibernate/id/enhanced/NoopOptimizer.java

```diff
@@ -38,12 +38,6 @@ public Serializable generate(AccessCallback callback) {
 		// The lastSourceValue field is only accessed by tests,
 		// so this is not a concern.
 		IntegralDataTypeHolder value = callback.getNextValue();
```

Comment:
The following change causes the following OptimizerUnitTest failures:
* testBasicNoOptimizerUsage
* testBasicNoOptimizerUsageWithNegativeValues

Both fail because the SourceMock intialValue is set to 0. After the following change,  NoopOptimizer#generate will return 0 as the first value (instead of 1).

What is the rationale for Hibernate forcing the first sequence value to be 1?